packages/chain-bridge/src/contracts/CrossChainRegistry.sol

```diff
@@ -766,6 +781,95 @@ contract CrossChainRegistry is ICrossChainRegistry, ProtocolAccessManaged {
         }
     }
 
+    /// @inheritdoc ICrossChainRegistry
+    function registerSourceChainRelationship(
```

Comment:
it's duplication of the `registerCrossChainRelationship` - so either have to specialized methods or one that's generic